src/script.cpp

```diff
@@ -41,7 +41,7 @@ bool Scripts::loadScripts(std::string folderName, bool isLib, bool reload)
 {
 	namespace fs = boost::filesystem;
```

Comment:
You can now change this to be `std::filesystem`, we used `boost` because C++11 didn't have `filesystem` but C++17 has.